### ISSUE TITLE
CSS detection of base attribute implementation support

### ISSUE THREAD

**Issue by gregwhitworth** [w3c/csswg-drafts#6195]
In issue #5998 we resolved to make `base` an attribute rather than a keyword. However, we also resolved that an author will want to be able to determine if `base` is supported to adjust their styles accordingly. I was actioned to open this issue to determine what that solution will look like.

@frivoal recommended a pseudo-class approach for detection since it will be added by element while an `@support` would be more generic and thus won't allow the granular detection.

**Comment by frivoal** [comment]
Right. To be able to style things properly, we need to detect the combination of:
* base styling is supported for this type of element
* base styling is set on this particular element

`@supports` might be extended to cover the first one, but it won't do the second one. Attribute selectors can be used to check on the second one, but not on the first one. Using the two in combination is clumsy, and error prone. So I think that a pseudo class is what we want.
